### PR TITLE
Pioneer BDP

### PR DESCRIPTION
**Pull request by Sami32**
Reported by IdPezzo in this post: http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=634
From this log: http://www.universalmediaserver.com/forum/download/file.php?id=362

New renderer found: "Pioneer BDP" with dlna details: {friendlyName=Pioneer BDP, address=192.168.0.15, udn=uuid:140479c0-58f3-1cef-84bf-745e1c896b95, manufacturer=PIONEER CORPORATION, modelName=BDP-170, modelNumber=1.0, modelDescription=IPI Media Renderer}

We was missing Pioneer BDP renderer in general.

### CODE REVIEW

**Comment by Sami32** [comment]
@Nadahar @SubJunk I know it's not incline on the left size, but the other photos that i found are bigger in width so i did this choice.

If you think i need to change it, just tell it.
About the x2 icon, i'm not sure that kind need it ?

**Comment by Nadahar** [comment]
@Sami32 The 2x icons are for OS X with Retina display. They will be displayed with the same size but with higher resolution.

File: src/main/external-resources/renderers/Pioneer-BDP.conf [hunk 1]
```diff
@@ -27,7 +27,6 @@ DLNALocalizationRequired = true
 TranscodeVideo = MPEGTS-H264-AC3
 TranscodeAudio = MP3
 DefaultVBVBufSize = true
-AutoExifRotate = true
 MediaInfo = true
```
Comment (Nadahar) [inline src/main/external-resources/renderers/Pioneer-BDP.conf]: 👍

**Comment by Nadahar** [comment]
There were some line ending issue confusing git when trying to rebase. I'm not sure exactly what happened, but I managed to squash it.

**Comment by valib** [comment]
👍

**Pull request merged**